packages/mdc-select/README.md

```diff
@@ -200,37 +200,11 @@ To disable a list item, set `aria-disabled` to `"true"`, and set `tabindex` to `
 </div>
 ```
 
-### Using the Pure CSS Select
```

Comment:
We should presumably also remove the tips/tricks section in this readme since it no longer applies (there is no longer a css-only MDC Select)